docs/book/src/topics/workload-identity.md

```diff
@@ -1,92 +1,86 @@
-    regardless because cloud provider azure integration is not yet done. The
-    steps are mentioned in the next section of workload cluster creation.
-  - The next list items links to steps on creating the federated
-    identity credentials. You will need to set up several environment
-    variables for each one:
-    - `SERVICE_ACCOUNT_NAMESPACE` : Namespace where the capz-manager and
-      azureserviceoperator-controller-manager pods will run.
-    - `SERVICE_ACCOUNT_NAME` : Name of the capz-manager or azureserviceoperator-default k8s service account.
-    - `SERVICE_ACCOUNT_ISSUER` : This is the path of the Azure storage
-      container which you created in the previous step which is:
-      `"https://${AZURE_STORAGE_ACCOUNT}.blob.core.windows.net/${AZURE_STORAGE_CONTAINER}/"`
-
-  - Create federated identity credentials for each of CAPZ and ASO using the steps outlined [here](https://azure.github.io/azure-workload-identity/docs/topics/federated-identity-credential.html)
-    You can either use `user-assigned-identity` or `AD application` to create federated identity credential and add `contributor` role to it.
-
-- Create a Kind cluster with necessary flags with the following command:
+  These environment variables will be used later, when creating the kind cluster.
 
-```bash
-cat <<EOF | kind create cluster --name azure-workload-identity --config=-
-kind: Cluster
-apiVersion: kind.x-k8s.io/v1alpha4
-nodes:
-- role: control-plane
-  extraMounts:
-    - hostPath: ${SERVICE_ACCOUNT_KEY_FILE}
-      containerPath: /etc/kubernetes/pki/sa.pub
-    - hostPath: ${SERVICE_ACCOUNT_SIGNING_KEY_FILE}
-      containerPath: /etc/kubernetes/pki/sa.key
-  kubeadmConfigPatches:
-  - |
-    kind: ClusterConfiguration
-    apiServer:
-      extraArgs:
-        service-account-issuer: ${SERVICE_ACCOUNT_ISSUER}
-        service-account-key-file: /etc/kubernetes/pki/sa.pub
-        service-account-signing-key-file: /etc/kubernetes/pki/sa.key
-    controllerManager:
-      extraArgs:
-        service-account-private-key-file: /etc/kubernetes/pki/sa.key
-EOF
-```
+- Create and upload a discovery document
 
-- Initialize a management cluster using `clusterctl` using the below command.
-  If you do not have `clusterctl` installed, then follow this [link](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl)
-  to install.
-```bash
-$ clusterctl init --infrastructure azure
-```
+  Create and upload a JWKS discovery document by following [these instructions](https://azure.github.io/azure-workload-identity/docs/installation/self-managed-clusters/oidc-issuer.html).
+
+- Create two federated identity credentials
+
+  Export environment variables used for creating the federated identity credentials:
+
+  - `SERVICE_ACCOUNT_NAMESPACE`: Namespace where the capz-manager and
+    azureserviceoperator-controller-manager pods will run.
+  - `SERVICE_ACCOUNT_NAME`: Name of the capz-manager or azureserviceoperator-default k8s service account.
```

Comment:
The namespace should be `capz-system` and the name is `capz-manager` and `azureserviceoperator-default` by default. I'm not sure where you could configure these to be different but there are obvious defaults. It would've saved me some time if I knew that these were the defaults as a first-time user.